src/Hl7.Fhir.Base/Serialization/engine/IFhirSerializationEngine.cs

```diff
@@ -39,12 +39,12 @@ public interface IFhirSerializationEngine
         /// </summary>
         /// <exception cref="DeserializationFailedException">Thrown when the deserializer encountered one or more errors in the FHIR Xml format.</exception>
         /// <returns>Null if the data did not contain a resource, but another FHIR datatype.</returns>
-        public Resource? DeserializeFromXml(string data);
+        public Base? DeserializeFromXml(string data);
```

Comment:
This is a breaking change, can we keep this change internal?  Otherwise, we'll need to add a comment to the PR that this is a breaking change.